Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:lts-alpine3.17
+
+WORKDIR /usr/src/app
+
+COPY package.json .
```

Comment:
There is no need for this line If you are already copying the entire app on line 9.

Also, Without a docker ignore file, line 9 will override the node_modules in the docker image with the node_modules file you have on your machine, Which makes the install on line 7 pointless locally at least.